src/Command/BaseCommand.php

```diff
@@ -15,28 +15,40 @@
 
 use Sonata\Doctrine\Model\ManagerInterface;
 use Sonata\MediaBundle\Provider\Pool;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 
 /**
  * This command can be used to re-generate the thumbnails for all uploaded medias.
  *
  * Useful if you have existing media content and added new formats.
  */
-abstract class BaseCommand extends ContainerAwareCommand
+abstract class BaseCommand extends Command
 {
     /**
-     * @return ManagerInterface
+     * @var ManagerInterface
      */
-    public function getMediaManager()
-    {
-        return $this->getContainer()->get('sonata.media.manager.media');
-    }
+    private $mediaManager;
 
     /**
-     * @return Pool
+     * @var Pool
      */
-    public function getMediaPool()
+    private $pool;
+
+    public function __construct(ManagerInterface $mediaManager, Pool $pool)
+    {
+        $this->mediaManager = $mediaManager;
+        $this->pool = $pool;
+
+        parent::__construct();
```

Comment:
is this needed?

Reply:
[It calls the `configure` method](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Console/Command/Command.php#L77).

Ref: https://symfony.com/doc/4.4/console.html#configuring-the-command